### PR TITLE
docs(blog): Add new RFC process in latest Gazette

### PR DESCRIPTION
**Pull request by sidharthachatterjee**

### CODE REVIEW

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 1]
```diff
@@ -156,6 +156,12 @@ We started off this year on a solid note with the release of [Gatsby Builds](/bl
 
 [Ward Peeters](https://twitter.com/wardpeet) built the new Gatsby Jobs API which enables Gatsby to better distribute CPU intensive work like image processing and HTML generation to workers on local machines and the Cloud.
 
+### New RFC Process
+
+[Dustin Schau](https://twitter.com/SchauDustin) proposed a new RFC process late last year. The main difference is that we'll be creating a new, top-level rfcs folder within the [gatsbyjs/gatsby](https://github.com/gatsbyjs/gatsby) monorepo and deprecate the old [gatsbyjs/rfcs](https://github.com/gatsbyjs/rfcs) repo.
```
Comment (laurieontech) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: Should this be `rfc` folder? Or RFCs folder?

Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: It's `rfcs` according to https://github.com/gatsbyjs/rfcs/blob/master/text/0011-move-rfcs-to-monorepo.md#rfcs-folder-in-the-monorepo

Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: Okay, I think I see what you mean after another read. It should probably be `rfcs` (as a code block I mean)

Comment (sidharthachatterjee) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]:
```suggestion
[Dustin Schau](https://twitter.com/SchauDustin) proposed a new RFC process late last year. The main difference is that we'll be creating a new, top-level `rfcs` folder within the [gatsbyjs/gatsby](https://github.com/gatsbyjs/gatsby) monorepo and deprecate the old [gatsbyjs/rfcs](https://github.com/gatsbyjs/rfcs) repo.
```

**Comment by laurieontech** [comment]
THanks Sid!

**Pull request merged**